graalpy-install-package/src/main/resources/META-INF/native-image/jni-config.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "name": "java.lang.Thread",
```

Comment:
I don't know why, but it seems that the agent is unable to detect this. Maybe, because it happens too early when the app starts or so.